hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -44,6 +44,21 @@ function fetch_remote_os_info {
     ssh -q "$ruser@$rhost" "cat /etc/os-release"
 }
 
+# Allow the condor user to run the WN client updater as the local users
+function setup_sudo_users {
+    users=$(get_mapped_users)
+    [[ -n $users ]] || errexit "Did not find any HTCondor-CE SCITOKENS user mappings"
+
```

Comment:
I think you'll actually want to take `$users` as an argument and return these lines to the 144-145:

1. The `$users` variable is used in other spots further down in the script
2. Generally, I think decisions to blow up should reside at the top-level (either explicitly or through exception handling). This is a little too hidden away for my liking.